code/game/machinery/doors/firedoor.dm

```diff
@@ -226,7 +226,7 @@
 	if(alarmed && density && lockdown && !allowed(user))
 		to_chat(user, "<span class='warning'>Access denied.  Please wait for authorities to arrive, or for the alert to clear.</span>")
 		return
-	else
+	else if(Adjacent(user))
```

Comment:
This also looks like a undocumented change that prevents AIs from interacting with the firedoors

Reply:
this just prevents the opening message from playing when the person isn't adjacent, otherwise it plays at the AI's core
![image](https://user-images.githubusercontent.com/22774890/98998348-b03e3900-253e-11eb-87d1-9c9be458736f.png)